src/mesh/mesh_io.jl

```diff
@@ -132,6 +132,36 @@ function save_mesh_file(mesh::UnstructuredQuadMesh, output_directory)
 end
 
 
+# Does not save the mesh itself to an HDF5 file. Instead saves important attributes
+# of the mesh, like its size and the type of boundary mapping function.
+# Then, within Trixi2Vtk, the P4estMesh and its node coordinates are reconstructured from
+# these attributes for plotting purposes
+function save_mesh_file(mesh::P4estMesh, output_directory)
+  # Create output directory (if it does not exist)
+  mkpath(output_directory)
+
+  filename = joinpath(output_directory, "mesh.h5")
+  p4est_filename = joinpath(output_directory, "p4est_file")
```

Comment:
Should this get an extension? Is it a binary or an ASCII file? If there are is "standard" extension, I would suggest to use just `p4est.dat`. Using `_file` seems redundant for a file...

Reply:
This is binary. I searched all source code of p4est to find an example file name, but I couldn't find one.

Reply:
Hm, ok. I'd leave off the `_file`, but about the extension I am not sure. I just feel like if you just call it `p4est`, it might be weird. Maybe `p4est_data` or `connectivity.p4est`? I don't know... just pick something you think looks good ;)

Reply:
>  I just feel like if you just call it `p4est`, it might be weird.

That's what I thought too when I appended `_file`, but I already expected one of you to complain.
I think `p4est_data` sounds pretty good.

Reply:
:+1: And hey, I'm not complaining, I am just pointing out something you already knew before... :sunglasses: